doomgrader.sh

```diff
@@ -2,15 +2,40 @@
 
 set -e
 
-# doomgrader config
+# doomgrader config; EDIT HERE!
 DOOMGRADER_ROOT=~/doomgrader
 STEAM_PATH=~/.steam/steam/steamapps/common
 
 # internal paths
 DOWNLOAD_PATH=$DOOMGRADER_ROOT/files
 DEPOTDOWNLOADER_PATH=$DOOMGRADER_ROOT/depotdownloader
 
-# prompt for steam credentials
+# function for copying files with prompt
+DOOMGRADER_COPY()
+{
+# copy game files to steam dir
+echo "Copying the game from $DOWNLOAD_PATH to $STEAM_PATH/DOOMEternal/"
+echo "THIS WILL OVERWRITE YOUR EXISTING DOOM ETERNAL GAME FILES!"
+read -p "Is this correct (y/n)?: " DOOMGRADER_ANSWER
+
+if [[ $DOOMGRADER_ANSWER == 'Y' || $DOOMGRADER_ANSWER == "yes" || $DOOMGRADER_ANSWER == "YES" || $DOOMGRADER_ANSWER == "y" || $DOOMGRADER_ANSWER == "Yes" ]]
+then
+    cp $DOWNLOAD_PATH/* $STEAM_PATH/DOOMEternal/ -rfv
```

Comment:
I'm not sure about adding `-v` here. It produces a lot of logs that are hard to read through. Also, please add back the `\cp`. It calls `/bin/cp` directly because some distro's alias it to require confirmation by default. Without it, some users may have to approve each file individually

Reply:
Okay, I will add back the \cp. Thanks for explaining what it does. 
I personally really like the -v. I don't like how the copying doesn't produce output, especially when the -rf is added. Can we keep it?

Reply:
Ok let's keep it then. The copy does take a while

Reply:
Thanks! I think it may help with settling fears of overwriting files from a script on the internet.